src/ssh.act

```diff
@@ -22,10 +21,14 @@ actor Client(cap: net.TCPConnectCap,
     # haha, this is really a pointer :P
     var _ssh_session: u64 = 0
 
+    def get_ssh_session():
+        return _ssh_session
+
```

Comment:
```suggestion
    action def get_affinity() -> u64:
        NotImplemented
```

then need to implement the necessary C call to return the actual affinity of the Client actor. We need this as action so the Channel actor can query the Client about affinity, so the Channel actor in turn can make sure it gets pinned to the same worker thread as the Client.